README.md

```diff
@@ -122,28 +129,32 @@ The interaction with OpenShift is via the command line tool _oc_ which is copied
            To login as administrator:
                oc login -u system:admin
 
-    Note that, the IP seen above is dynamic and can change. It can be retrieved with `minishift ip`. Also,
-    instead of 'kvm', you will see 'xhyve' on Mac OS and 'hyperv' on Windows.
-
-    **Note:** By default Minishift uses the driver most relevant to the host OS.
-To use a driver of choice for Minishift, use the `--vm-driver=xxx` flag with `minishift start`. For example, to use VirtualBox instead of KVM for Fedora, use `minishift start --vm-driver=virtualbox`.
+  **Note**:
 
-1. Add `oc` binary to the _PATH_:
+  - The IP is dynamically generated for each Minishift instance. To check the IP, run the `minishift ip` command.
+  - By default Minishift uses the driver most relevant to the host OS. To use a different driver, set the `--vm-driver` flag in `minishift start`. For example, to use VirtualBox instead of KVM on Linux operating systems, run `minishift start --vm-driver=virtualbox`. For more information about `minishift start` options,
+  see the [minishift start command reference](/docs/minishift_start.md).
 
-     **Note:** How to modify the _PATH_ varies depending on host OS, version of the OC binary and
-     other variables. In case of doubt, you can check the content of the
-     `~/.minishift/cache/oc` directory.
+2. Add the `oc` binary to the _PATH_ environment variable. For example, on Linux
+operating systems, run the following command:
```

Comment:
The first step says its a Linux example, so  'For example on Linux operating systems' again here, seems unnecessary, specially when it is followed by a note saying it could be different for other Operating systems.